src/main/java/com/playmonumenta/networkchat/ChatFilter.java

```diff
@@ -177,7 +178,9 @@ public ChatFilterPattern command(@Nullable String command) {
 		}
 
 		public void run(CommandSender sender, final ChatFilterResult filterResult) {
+			assert NetworkChatPlugin.getInstance() != null;
```

Comment:
Asserts are bad IMO in production code. Yes, this should never resolve to true, and if it does, there's a problem. But this will crash the entire server, whereas a null pointer exception would not.